### DISCUSSION TITLE
Hey Can Some one help me, I want to remove the above bar completely which includes logo, company name, search and bar.

### DISCUSSION THREAD

**Discussion by RohanSagar11** [squidfunk/mkdocs-material#6805] [Q&A] [answered]
I want this thing to be removed any yml feature actually this thing comes under the body, Any Suggestions->

<img width="940" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/56624624/dca6113c-be53-4870-bcdf-edfb9b7a1b2f">

**Comment by alexvoss** [comment, marked as answer]
There is no configuration for removing the header. This is not something many people need and we can only make so many things configurable without making the code unmaintainable. You are looking, therefore, for a [customization](https://squidfunk.github.io/mkdocs-material/customization/). If you create your own `main.html` and [override the `header`block](https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks) to be empty then you should be good.

**Reply by kamilkrzyskow** [reply]
An empty `header` block will also remove the `navigation.tabs`, so if you want to keep them, then instead of the empty block, you need to override the `partials/header.html` template and remove everything apart of the tabs logic at the bottom.

**Reply by alexvoss** [reply]
Ah, dang, I did not see that. To avoid editing partials (maintenance issue), perhaps change what the `header` block includes to include the `tabs` partial directly? Might be worth a try, though not sure if the CSS will still work, depends a bit on how it uses selectors.
